src/processor/tree/mod.rs

```diff
@@ -49,7 +55,10 @@ impl TreeProcessor {
 #[async_trait]
 impl Processor for TreeProcessor {
     async fn run(mut self, mut rx: mpsc::Receiver<CommitBlock>) {
+        let mut metric = PerfMetric::new("snapshot");
```

Comment:
```suggestion
        let mut metric = PerfMetric::new("tree_insert");
```
This would also be computing the time it takes to insert another block into the state tree, right? We should split this in two in that case: one for `tree_insert` and one for `snapshot`.

Reply:
OK